user/user_test.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"github.com/TempleEight/spec-golang/user/dao"
```

Comment:
This been gofmt'ed?

Usually custom imports get sorted to after the standard ones